_layouts/security.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: base
+---
+
+<div class="security-page grid-wrapper">
+    <div class="grid__item width-12-12">
+        <h1>Reporting of CVEs and Security Issues</h1>
+        <h2 class="page-subtitle">The Wild<strong>Fly</strong> community and our sponsor, Red Hat, take security bugs very
+            seriously
+        </h2>
+        <p>We aim to take immediate action to address serious security-related problems that involve our projects.
+            Note that we will only fix such issues in the most recent major release of Wild<strong>Fly</strong>.</p>
```

Comment:
I think "major" should be minor; e.g. we would not have fixed 26.0 once 26.1 was out.